### DISCUSSION TITLE
Winsock

### DISCUSSION THREAD

**Discussion by JohnLamontagne1996** [twinbasic/twinbasic#287] [Q&A] [answered]
Will The Winsock component be reimplemented in Twin Basic, so Vb6 projects using it will work fine in Twin Basic?
![WinSock](https://user-images.githubusercontent.com/87338545/126543895-7d822964-8ff8-4ed1-b5e1-32901c2a8fa2.png)

**Comment by Kr00l** [comment]
tB will support it. So you can just put the MS winsock on a tB form like you did with VB6.
A direct replacement is not needed. Though you are restricted then to 32 bit I guess.

**Comment by WaynePhillipsEA** [comment, marked as answer]
Yes, this will definitely be supported.  It's just a wrapper around the winsock APIs provided by the OS, so it will be easy to recreate (with both 32 and 64 bit support)
